tools/build/WindowsInstaller/Delete.bat

```diff
@@ -88,7 +88,6 @@ del Qt5WebViewd.dll
 del Qt5Widgetsd.dll
 del Qt5WinExtrasd.dll
 del Qt5Xmld.dll
-del Qt5XmlPatternsd.dll
```

Comment:
this should probably kept, removing this line means the installer would include the debug dll for qt5xmlpattern, which would be useless